Source/common/SNTXPCControlInterface.h

```diff
@@ -46,6 +50,12 @@
 - (void)databaseRuleForBinarySHA256:(NSString *)binarySHA256
                   certificateSHA256:(NSString *)certificateSHA256
                               reply:(void (^)(SNTRule *))reply;
+///
+///  Decision ops
+///
+- (void)decisionForFilePath:(NSString *)filePath
```

Comment:
We need the SNTFileInfo object for "Scope" processing.
We could look into creating, yet another, wrapper object to store the hash, certHash and other relevant SNTFileInfo properties. Making that object secure coding compliant for XPC travels.
Or take the time to make SNTFileInfo secure coding compliant :/